modules/installation-special-config-storage.adoc

```diff
@@ -30,9 +30,13 @@ When the TPM v2 or Tang encryption modes are enabled, the {op-system} boot disks
 
 This feature:
 
-* Is available for installer-provisioned infrastructure and user-provisioned infrastructure deployments
+* Is available for installer-provisioned infrastructure, user-provisioned infrastructure, and Assisted Installer deployments
```

Comment:
It would be worth mentioning that encryption can be enabled on all nodes, or only on master / workers. See:

https://github.com/openshift/assisted-service/blob/94f729c27415ed6887a0a5e04ab602f57641af68/swagger.yaml#L5120-L5124